README.rst

```diff
@@ -43,9 +43,10 @@ Retrieve User Profile
         response = facebook.get(endpoint='/me?fields=id,name')
     except FacebookResponseException as e:
         print(e.message)
-    else:
-        print('User id: %(name)s' % {'name': response.json_body.get('id')})
-        print('User name: %(name)s' % {'name': response.json_body.get('name')})
+     print('User id: %(id)s, name: %(name)s'.format({
+        'id': response.json_body.get('id'),
```

Comment:
This will not do a string replacemente. `format` should be used as:
```     
print('User id: {id}, name: {name}'.format(
    id=response.json_body.get('id'),
    name=response.json_body.get('name'), 
)
```